styles.css

```diff
@@ -0,0 +1,18 @@
+body {
+  font-family: Arial, sans-serif;
+}
+
+.login-page {
+  display: flex;
+}
+
+.login-image {
+  flex: 60%;
```

Comment:
The image path is hardcoded in the styles.css file. Consider making this dynamic so that it can be easily changed or updated.